docs/releases.rst

```diff
@@ -25,6 +25,8 @@ the given job group.
 The ``note`` field of tmt :ref:`/spec/plans/results` changes from
 a string to a list of strings, to better accommodate multiple notes.
 
+New subpackage ``tmt+ansible`` introduced. It covers prepare step
+with ``--how ansible`` option by installing ansible requirements.
```

Comment:
* Should mention other Ansible-dependant plugins too, `prepare/feature`.
* Do not use command-line `--how`, this applies to plugins and let's assume users know how to select a plugin :)

How about the following?

```
Introducing new subpackage ``tmt+ansible``, to pull in Ansible requirements for plugins
that use Ansible under the hood, :ref:`/plugins/prepare/ansible`,:ref:`/plugins/finish/ansible`,
and :ref:`/plugins/prepare/feature`.
```